src/control/lib/hardware/sysfs/provider.go

```diff
@@ -216,6 +248,30 @@ func (s *Provider) getPCIAddress(path string) (*hardware.PCIAddress, error) {
 	return nil, errors.Errorf("unable to parse PCI address from %q", path)
 }
 
+func (s *Provider) addNetvscDevice(topo *hardware.Topology, path string) error {
+	ifaceName := filepath.Base(path)
+
+	virt := &hardware.VirtualDevice{
+		Name: ifaceName,
+		Type: hardware.DeviceTypeNetInterface,
+	}
+
+	backingDev, err := s.getBackingDevice(path, topo.AllDevices())
+	if err != nil {
+		s.log.Noticef("Skipping NetVSC network adapter %q: not found physical backing device",
```

Comment:
nit: suggest changing text from "not found  physical backing device" to "physical backing device not found"

Reply:
- [ ] update informative message

Reply:
Fixed with commit 8101ff4fad5b7c3a01585614931840bc0481bcb5